docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,59 @@
 
 TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
 
+== OpenShift 4.11
+
+OpenShift version 4.11 is available.
+Find the release notes at the upstream documentation as https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html[OpenShift Container Platform 4.11 release notes].
+The https://cloud.redhat.com/blog/whats-new-in-red-hat-openshift-4.11[What’s New in Red Hat OpenShift 4.11] blog post can also be a valuable resource.
+
+More flexible alert routing for User Workload Monitoring::
+
+You can now https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-alert-routing-for-user-workload-momitoring[enable alert routing] as well as spin up a https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-enable-dedicated-alertmanager-for-user-defined-alerts[dedicated Alertmanager] for User Workload Monitoring.
+
+Disk usage base metrics retention::
+
+Metrics retention can now be defined based on https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-confgure-retention-size-for-metrics-storage[disk usage].
+
+New subcommand to work with tokens::
+
+`oc create token` replaces https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-oc-commands-flags-tokens-deprecated[several existing commands].
+Check your aliases, scripts and other convenience tools you use.
+
+Deprecation of `snapshot.storage.k8s.io/v1beta1`::
+
+Before update a cluster, check for usage of `snapshot.storage.k8s.io/v1beta1`.
+If used, inform the affected users and ask them to update to `snapshot.storage.k8s.io/v1`.
+
+...
```

Comment:
What's that?

Reply:
It is called a thematic break: https://docs.asciidoctor.org/asciidoc/latest/blocks/breaks/#thematic-breaks.

I used it here to clarify that the reference to the blog post does not belong to the above section, but is a remark for the content that follows. It is rendered as a horizontal line (html <hr />).